docs/source/pika-comparison.rst

```diff
@@ -0,0 +1,288 @@
+
+.. _Pika: https://pika.readthedocs.io/en/stable/
+.. _RabbitMQ tutorials: https://www.rabbitmq.com/getstarted.html
+.. _work queues: https://www.rabbitmq.com/tutorials/tutorial-two-python.html
+.. _topics: https://www.rabbitmq.com/tutorials/tutorial-five-python.html
+.. _RPC: https://www.rabbitmq.com/tutorials/tutorial-six-python.html
+.. _aio-pika: https://aio-pika.readthedocs.io/en/latest/index.html
+
+Coming from Pika
+================
+
+KiwiPy comes natively with a RabbitMQ communicator (others can be added by extending the :py:class:`~kiwipy.Communicator` interface) and thus it may be useful to see how to achieve the same things in `Pika`_ (the standard library used in the `RabbitMQ tutorials`_) and kiwiPy.  This also shows some of the differences, particularly where kiwiPy is less verbose by setting sensible defaults for the user.
```

Comment:
It is useful to keep each sentence on a separate line. This works nicer with git since it is line-based. But up to you